[WIP] User can view weather

GergKllai1 opened the pull request:
# PT Link

[User can view weather](https://www.pivotaltracker.com/story/show/163087012)

# User Story

```
As a user
In order to see news about the weather conditions
I would like to see weather information for my location in the navbar
```

# Screenshots

![screen shot 2019-01-21 at 11 48 44](https://user-images.githubusercontent.com/43814981/51469830-8cf73800-1d72-11e9-9a23-5596bec653d3.png)


# Description of PR

- Uses geocoder gem to fetch location using IP address, sets Stockholm as default
- Uses the freecodecamp weather api to fetch weather data based on geolocation
- Outputs current/min/max temp city and a weather icon

GergKllai1 committed:
Adds geocoder gem for getting location based on current IP, adds the method to get the weather info from freecode weather api, hardcoded as of now

kianaditya committed:
Merge branch 'weather' of github.com:GergKllai1/CA_Newsroom_team_1 into weather

kianaditya committed:
refactors get_weather function

GergKllai1 committed:
Moves get_weather method to application controller, outputs data in the navbar, adds logo and current city

tochman commented:
Merge conflicts 🤪

tochman commented:
Please make sure to fix the conflicts and we can take a look at testing API content.

kianaditya committed:
Merge branch 'develop' of github.com:CraftAcademy/CA_Newsroom_team_1 into weather

kianaditya committed:
Fixes merge conflicts

tochman committed:
adds feature for viewing weather info
adds webmock gem
mocks out api response in test env

tochman committed:
configures cucumber setup to allow chrome varsion check

tochman committed:
clear duplicaded code from cucumber env file

tochman committed:
extracts get weather method to a service

tochman committed:
managed to find a way to retrieve cookies in Selenium

tochman committed:
got the cookie read working in test

tochman committed:
recreates the db

tochman committed:
merges with upstream and fixes merge conflicts
rearranges scenarios in user can vew articles feature to get rid of the AS failures

tochman committed:
WIP cookie assertion

tochman committed:
comments out cookie assertion. Will return to this at later stage
